modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -16,11 +16,6 @@ def submit
 
           raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if bgs_claim.blank?
 
-          if dependent_service(bgs_claim).dependent_type_claim? && params[:sponsorIcn].blank?
-            claim_type = bgs_claim&.dig(:benefit_claim_details_dto, :bnft_claim_type_cd)
-            detail = "SponsorICN is required for claim type #{claim_type}"
-            raise ::Common::Exceptions::ResourceNotFound.new(detail:)
-          end
           file_number_check(icn: params[:sponsorIcn])
```

Comment:
Does this line (line 24:19) need to be removed as well? 

`file_number_check(icn: params[:sponsorIcn])` since `sponsorIcn` would not be there